inst/extdata/common/iso_GCAM_regID.csv

```diff
@@ -1,244 +1,244 @@
-# File:iso_GCAM_regID.csv,,,
```

Comment:
Looks like it got Windows line endings, which are mostly harmless, but we should revert this change anyhow.